### ISSUE TITLE
not able to overide prometheus.metricServer.port 

### ISSUE THREAD

**Issue by swethagannapu** [kedacore/charts#490]
No matter what i try I am not able to override prometheus.metricServer.port 
 helm upgrade --install keda kedacore/keda --namespace kube-system --version 2.10.2 \
--set metricsServer.useHostNetwork=true --set "prometheus.metricServer.port=8082"

```
Containers:
  keda-operator-metrics-apiserver:
    Image:           ghcr.io/kedacore/keda-metrics-apiserver:2.10.1
    Ports:           2443/TCP, 8080/TCP
    Host Ports:      2443/TCP, 8080/TCP
    SeccompProfile:  RuntimeDefault
    Args:
      /usr/local/bin/keda-adapter
      --port=8080
      --secure-port=2443
```

Is there anything that i am missing here?

**Comment by tomkerkhove** [comment]
Can you do a helm template and share the output please?

**Comment by emctl** [comment]
Looks to be fixed in latest chart (2.11.0)
